pydicer/visualise/data.py

```diff
@@ -113,12 +120,25 @@ def visualise(self, patient=None):
 
                     img_path = Path(img_row.path)
 
-                    img = sitk.ReadImage(str(img_path.joinpath(f"{img_row.modality}.nii.gz")))
+                    # save image inside structure directory
```

Comment:
I agree for sure, but for the time being I might leave this one. In the future it would be nice to provide the functionality to the user to easily load the data as SimpleITK image objects, but needs a bit more thought around exactly how that should work. But once we do that we can refactor out this sort of code.